### PR TITLE
Format JSON output of cli commands to be human readable

### PR DESCRIPTION
**Pull request by silhouettes**

### CODE REVIEW

File: cli/script/command-executor.ts [hunk 1]
```diff
@@ -582,6 +582,10 @@ function printDeploymentList(command: cli.IDeploymentListCommand, deployments: D
     }
 }
 
+function printJson(object: any) {
```
Comment (dtivel) [inline cli/script/command-executor.ts]: : void

File: cli/test/cli.ts [hunk 2]
```diff
@@ -7,6 +7,11 @@ import * as cli from "../definitions/cli";
 import * as cmdexec from "../script/command-executor";
 import * as os from "os";
 
+function assertJsonDescribesObject(json: string, object: Object) {
```
Comment (dtivel) [inline cli/test/cli.ts]: : void

**Comment by dtivel** [comment]
looks good

**Comment by silhouettes** [comment]
Per prior discussion, I've also removed the word 'minimum' used to describe app store versions in the CLI, as we match the latest update by exact app store version. Thanks dtivel and lostintangent!

**Pull request merged**